contrib/seccomp/seccomp_default.go

```diff
@@ -585,7 +585,6 @@ func DefaultProfile(sp *specs.Spec) *specs.LinuxSeccomp {
 					"mount",
 					"mount_setattr",
 					"move_mount",
-					"name_to_handle_at",
```

Comment:
Actually; wondering now if it should be added to the allow-list instead, otherwise it may get denied by default? See https://github.com/moby/moby/pull/45766/files#r1244845677

Reply:
That's right; PR has been updated.